spec/features/state_file/complete_intake_spec.rb

```diff
@@ -165,7 +165,7 @@
       expect(page).to have_text I18n.t('state_file.questions.terms_and_conditions.edit.title')
       click_on I18n.t("state_file.questions.terms_and_conditions.edit.accept")
 
-      step_through_df_data_transfer
+      step_through_df_data_transfer("Transfer AZ Old sample")
```

Comment:
Are you using the string for label purposes?

Reply:
The string is actually used to select the button to click in the test. This was hardcoded to 
```click_on "Transfer my 2023 federal tax return to FileYourStateTaxes"```
 so I added an optional parameter to allow clicking on one of the other elements instead. (See state_file_intake_helper.rb Line 70